### ISSUE TITLE
Need to be able to distinguish visually hyperlinks from text on the front page

### ISSUE THREAD

**Issue by fedorov** [ImagingDataCommons/IDC-WebApp#556]
Would it be possible to change the styling of the hyperlinks for the portal front page to indicate what text is hyperlinked? As is, it is impossible to know if certain text is hyperlinked or not, until you mouse over.

![2021-02-24_12-29-01](https://user-images.githubusercontent.com/313942/109057273-894fe100-76af-11eb-9aa4-048a29049cb0.gif)

It would be very helpful if hyperlinks were distinguished by underline, highlight, different color, bold, whatever.

**Comment by G-White-ISB** [comment]
Call out for opinions - should the hyperlinks be bold, underlined, etc?

**Comment by fedorov** [comment]
Making them underlined in all places would make the site consistent with itself.

**Comment by pieper** [comment]
Yes, a consistent underlining of links would be a web-standard approach.

**Comment by s-paquette** [comment]
The main issue is, due to the number of links on the site, it really makes the site look ungainly. Especially the Navbar.

Note that none of the other sites use underlining on their links:
https://proteomic.datacommons.cancer.gov/pdc/
https://gdc.cancer.gov/access-data/data-access-processes-and-tools

There's a reason for this: it's visually ugly any time you have a large number of links, and in any kind of menu.

All of our links underline when hovered. We can bold them, or make them another color, but in general underlining makes for intense page clutter on page like ours. It's not required by 508 - all that's required is that they be visually distinct, and not rely on only color. 

If nothing else, doing this will make our site look significantly different than the other Commons.

**Comment by pieper** [comment]
True, underlining isn't really the issue, just visually distinct and consistent on all our pages is the key issues.  Good idea to try to maintain visual consistency with other data commons nodes.

**Comment by fedorov** [comment]
I have nothing to say other than restate the title of the issue: "Need to be able to distinguish visually hyperlinks from text on the front page"

**Comment by fedorov** [comment]
Curiously, PDC does not underscore the links, GDC portal does. cancer.gov and DataCommons pages do not underline the hyperlinks, so I guess this means we should just make the "Account" page appearance consistent with the front page.
